src/su/cli.rs

```diff
@@ -242,7 +242,7 @@ impl SuOptions {
                     }
                 }
             } else {
-                // when no option is provided (styarting with - or --) the next argument is interpreted as an username
+                // when no option is provided (styarting with - or --) the next argument is interpreted as a username
```

Comment:
```suggestion
                // when no option is provided (starting with - or --) the next argument is interpreted as a username
```
I am pretty sure "starting" is not in the dictionary, too 😄

Reply:
Its like partying when you start :wink: